Sources/ViaSwiftUtils/UIKit/UIImageView+Rotation.swift

```diff
@@ -43,26 +45,26 @@ public extension UIView {
     }
     
     final var isRotating: Bool {
-        return layer.animation(forKey: viewRotationAnimationKey) != nil
+        return layer.animation(forKey: RotationAnimation.key) != nil
```

Comment:
You don't need explicit 'return' with the new Xcode :P, Maybe you can add a new rule for it.

Reply:
Yes, great spot! There already is an `implicit_return` rule, which I enabled in this very PR. But as for the current latest version - 0.38.0, this rule only has a default mode which works for the returns in the **closures**.

An extension to this rule is already under active development: https://github.com/realm/SwiftLint/issues/2870

I expect that `implicit_return` will get 2 new modes - "function" and "getter" maybe as soon as in the next version. I will make sure they are enabled as well. 😄